.travis.yml

```diff
@@ -27,8 +27,8 @@ cache:
 env:
   matrix:
     # Test builds first, since they don't upload things.
-    - TEST_GOAL=coverage
-    #- TEST_GOAL=performance
+    - TEST_GOAL=coverage TOXCORE=toktok/c-toxcore
+    - TEST_GOAL=performance TOXCORE=irungentoo/toxcore
```

Comment:
Ok. Looks like you don't care much about the actual "TEST_GOAL"s then, but that it compiles.